common/src/main/java/com/microsoft/identity/common/internal/controllers/CommandDispatcher.java

```diff
@@ -476,7 +488,15 @@ private static boolean eligibleToCache(CommandResult commandResult) {
      * @return
      */
     private static boolean eligibleToCacheException(BaseException exception) {
-        if (exception instanceof IntuneAppProtectionPolicyRequiredException) {
+        final String mErrorCode;
+        if (exception instanceof BrokerCommunicationException) {
+            mErrorCode = ((BrokerCommunicationException) exception).getCategory().toString();
```

Comment:
`category` strictly cannot be `null`? This is certain?

Reply:
It is dereferenced in the sole constructor of the class.

Reply:
Actually, we don't need to do this at all... BrokerCommunicationException /is a/ BaseException, and the errorCode is category.toString().

Reply:
I'm inclined to leave it alone, though.